ghost/core/core/frontend/helpers/ghost_head.js

```diff
@@ -212,6 +220,37 @@ module.exports = async function ghost_head(options) { // eslint-disable-line cam
     const favicon = blogIcon.getIconUrl();
     const iconType = blogIcon.getIconType(favicon);
 
+    // loop over the head_excludes array and set the isUnwanted object to true for each item
+    if (options.data.config?.head_excludes) {
+        options.data.config.head_excludes.forEach((item) => {
+            isUnwanted[item] = true;
+        });
+    }
```

Comment:
```suggestion
    const excludeList = new Set(options.data.config?.head_excludes ?? [])
```

Reply:
Thank you so much for your help here and on Slack!  Think I've got it all fixed up now!